docs/04-operation-guides/operations/smsh-01-istio-enable-sidecar-injection.md

```diff
@@ -12,9 +12,9 @@ You can enable sidecar proxy injection for either an entire Namespace or a singl
    kubectl label namespace {YOUR_NAMESPACE} istio-injection=enabled
    ```
 
-* To enable sidecar proxy injection for a Deployment, add this to the Deployment configuration file as either a label or an annotation: `sidecar.istio.io/inject: "true"`
+* To enable sidecar proxy injection for a Deployment, add this label to the Deployment configuration file: `sidecar.istio.io/inject: "true"`
 
-Note that the Namespace label takes precedence over the Pod label or annotation.
+Note that the Namespace label takes precedence over the Pod label.
```

Comment:
```suggestion
Note that if any of Namespace or Pod label disables Istio injection, the sidecar proxy will not be injected.
```

Reply:
That should be the case if I'm looking correctly
| default policy (enableNamespacesByDefault) | namespaceSelector (label: "istio-injection"]) | Pod label(sidecar.istio.io/inject) | Sidecar injected? |
|:------------------------------------------:|:---------------------------------------------:|:-------------------------------------------------:|:-----------------:|
|            true	            		             |                 enabled						                 |                    true 						                    |      yes				      |
|                  true			                   |                 enabled						                 |                    false						                    |      no				       |
|                true     		                 |                disabled						                 |                    true						                     |      no				       |
|                  true			                   |                disabled						                 |                    false						                    |      no				       |
|                  true			                   |                  not present                  |                    true						                     |      yes				      |
|                  true			                   |                not present				                |                    false						                    |      no				       |
|                  true			                   |                not present				                |                  not present 			                  |      yes				      |
|                 false   		                 |                 enabled						                 |                    true 						                    |      yes				      |
|                 false   		                 |                 enabled						                 |                    false						                    |      no				       |
|                 false   		                 |                disabled						                 |                    true						                     |      no				       | 
|                 false   		                 |                disabled						                 |                    false						                    |      no				       |
|                 false   		                 |                not present				                |                    true						                     |      yes				      |
|                 false   		                 |                not present				                |                    false						                    |      no				       |
|                 false   		                 |                not present				                |                  not present			                   |      no				       |

Reply:
Could you provide the link to the docs containing that table? I think we could link it as a further reference